moveit_plugins/moveit_fake_controller_manager/src/moveit_fake_controller_manager.cpp

```diff
@@ -35,167 +35,173 @@
 /* Author: Ioan Sucan, Dave Coleman, Robert Haschke */
 
 #include "moveit_fake_controllers.h"
-#include <ros/ros.h>
 #include <moveit/robot_state/robot_state.h>
 #include <moveit/robot_model_loader/robot_model_loader.h>
-#include <sensor_msgs/JointState.h>
+#include <sensor_msgs/msg/joint_state.hpp>
 #include <pluginlib/class_list_macros.hpp>
-#include <ros/console.h>
 #include <map>
 #include <iterator>
 
 namespace moveit_fake_controller_manager
 {
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit.plugins.moveit_fake_controller_manager");
 static const std::string DEFAULT_TYPE = "interpolate";
 static const std::string ROBOT_DESCRIPTION = "robot_description";
 
 class MoveItFakeControllerManager : public moveit_controller_manager::MoveItControllerManager
 {
 public:
-  MoveItFakeControllerManager() : node_handle_("~")
+  MoveItFakeControllerManager() : MoveItControllerManager()
   {
-    if (!node_handle_.hasParam("controller_list"))
+  }
+
+  void initialize(const rclcpp::Node::SharedPtr& node)
+  {
+    // TODO(henningkayser): use flexible base
```

Comment:
Do we need to address this TODO here? I would imagine that a variable namespace is configured using the node interfaces.

Reply:
I'm not sure what you meant by flexible base, so I left it alone.

Reply:
Yeah, I think that was related to adding support for configs outside of the private namespace. Since this is not an issue right now I would just remove this TODO. however, I would make the PARAM_BASE_NAME a `constexpr char PARAM_BASE_NAME[]` and make it global in this namespace.

Reply:
remove todo 7a6b31d